news/2021-06-20-mappers-report-may.md

```diff
@@ -0,0 +1,160 @@
+
+Other than being a hard rock mapper himself, KuroRev has also been endeavouring to push his first full solo set with the lovely future bass song [YUC'e - Cinderella Syndrome](https://osu.ppy.sh/beatmapsets/1467712), showcasing his mastery on movement and immaculate pattern structure. If you do like his maps, consider clicking that bell on his profile to not miss out on his later bangers. We are eager to see his motivation grow further and hopefully see him create more interesting maps!
+
+## Mapping Contests
+
+Another month of community-hosted contests is upon us, this time with something sweet for all osu!taiko and osu!catch mappers with some really great prizes. Don't know how to map either mode? That doesn't matter — you should try your luck anyway! Who knows, you may discover some hidden master mapper talent hiding within you.
+
+### osu!flushed Taiko Mapping Contest
+
+The [osu!flushed Taiko Mapping Contest](https://osu.ppy.sh/community/forums/topics/1342681) is a brand new contest where the responsibility of choosing the song falls onto your hands. That's right, you will have an *almost* complete freedom in song choice, where the only restriction is that it needs to follow a certain theme.
+
+As for the prizes, the winner gets 6 months of osu!supporter, while the runners-ups of 2nd and 3rd place receive 4 months of osu!supporter.
+
+Don't wait for the drums to roll - go ahead and sign yourself up via the thread above to show who places the blues and reds the best!
+
+### osu!catch 2021 Summer Featured Artist Cup
+
+As the summer cracks the door open, the fruits start ripening. To fill your cup with the cream of the crop, be sure to join the [osu!catch Featured Artist Cup](https://osu.ppy.sh/community/forums/topics/1327798) right away!
+
+Featuring songs from our [Featured Artists](https://osu.ppy.sh/beatmaps/artists) who joined the lineup between March 20th - June 20th, this contest will test your mapping capabilities in two difficulties: one being Platter or lower, and the other being Rain or higher.
+
+But that's not all - this one has some hefty prizes too! 
+
+The winner will be rewarded with 100 USD in cash, plus having their set modded by the host and the judges! Should you not make it to the top, the runners-up will still get 20 USD in cash in addition to 4 months of osu!supporter for 2nd place, and the 3rd place will enjoy 2 months of osu!supporter.
+
+Have we got you interested in getting your hands dirty with the harvest? Then be quick - the sign-ups are closing soon on **June 20th**!
```

Comment:
```suggestion
Have we got you interested in getting your hands dirty with the harvest? Then be quick — the sign-ups are closing soon on **June 20th**!
```

Reply:
That sentences got completely reworded as the deadline sadly passed